plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -399,10 +406,19 @@ public static Transaction newCreateTableTransaction(TrinoCatalog catalog, Connec
         String targetPath = getTableLocation(tableMetadata.getProperties())
                 .orElseGet(() -> catalog.defaultTableLocation(session, schemaTableName));
 
-        ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builderWithExpectedSize(2);
+        ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builderWithExpectedSize(4);
```

Comment:
You can take out the expected size
```suggestion
        ImmutableMap.Builder<String, String> propertiesBuilder = ImmutableMap.builder();
```

Reply:
Done.